src/Autocomplete.test.js

```diff
@@ -86,39 +84,6 @@ describe('Autocomplete', () => {
       })
     })
 
-    const expectedSuggestions = [
-      {
-        label: '123 Sesame St New York NY',
-        value: {
-          city: 'New York',
-          primary_line: '123 Sesame St',
-          state: 'NY',
-          zip_code: '12345'
-        }
-      },
-      {
-        label: "123 Bowser's Castle Mushroom Kingdom JA",
-        value: {
-          city: 'Mushroom Kingdom',
-          primary_line: "123 Bowser's Castle",
-          state: 'JA',
-          zip_code: '12345'
-        }
-      },
-      {
-        label: "123 Micky's Clubhouse Disneyland FL",
-        value: {
-          city: 'Disneyland',
-          primary_line: "123 Micky's Clubhouse",
-          state: 'FL',
-          zip_code: '12345'
-        }
-      }
-    ]
-
-    expect(handleSuggestion).toHaveBeenCalled()
-    expect(handleSuggestion).toHaveBeenCalledWith(expectedSuggestions)
-
```

Comment:
What's the reason for removing this test scenario? If it's a temporary bug let's comment this out instead

Reply:
We removed the onSuggestion prop in the react component